src/Forms.js

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from "react";
+import "./Forms.css";
+
+const Forms = ({ onClose, onSubmit, formType, content, setContent }) => {
```

Comment:
Closer, but I think you've gone too general here. Let's pair on this and I'll show you a happy middle-ground.